packages/hurl/src/http/mod.rs

```diff
@@ -59,7 +59,7 @@ mod response;
 mod response_cookie;
 mod response_debug;
 mod response_decoding;
-mod response_ip;
+mod ip;
```

Comment:
`mod ip` could be up a few lines so all the mod are sorted. You can reformat the code with `cargo fmt` and the CI/CD will be appeased.